float64 current   # 現在電流
float64 velocity  # 現在速度
float64 position  # 現在位置
uint8 error_code  # エラーコード